system-addon/lib/TopSitesFeed.jsm

```diff
@@ -65,9 +69,15 @@ this.TopSitesFeed = class TopSitesFeed {
       }
     }
 
-    // Now, get a screenshot for every item
+    // Now, get a tippy top icon or screenshot for every item
     for (let link of links) {
       if (!link) { continue; }
+
+      // Check for tippy top icon.
+      link = this._tippyTopProvider.processSite(link);
```

Comment:
Yeah, figured it should just should just move on if it isn't fully init'd instead of waiting. I guess we would want to dispatch an action when init is done to refresh again. Maybe there should be a tippy top feed like you suggested 🤔